src/bioregistry/data/bioregistry.json

```diff
@@ -59265,6 +59265,37 @@
       "uri_format": "http://nikkajiweb.jst.go.jp/nikkaji_web/pages/top_e.jsp?CONTENT=syosai&SN=$1"
     }
   },
+  "jecfa": {
+    "contact": {
+      "email": "jecfa@who.int",
+      "name": "JECFA Secretariat at WHO"
```

Comment:
Is it at all possible to identify a specific person who could serve as contact?

Reply:
https://www.who.int/news-room/articles-detail/call-for-data-jecfa-95-meeting

This page lists the contact name for jecfa@who.int as Kim Petersen.

Reply:
I found kpetersen@who.int on a publication as a specific email address. Don't see an ORCID though.